src/styles/_common.scss

```diff
@@ -162,3 +162,12 @@ $logos: VNG JD-Cloud---AI Zhihu Xiaomi BookMyShow Shopee Meituan-Dianping iQiyi
     opacity: 1 !important;
   }
 }
+
+@mixin mobile-section-padding {
```

Comment:
The `.section` has been defined in [global.scss](https://github.com/pingcap/website/blob/master/src/styles/global.scss#L207). You can remove this duplicate definition here.